lib/buildpackutil.py

```diff
@@ -53,6 +53,18 @@ def get_database_config(development_mode=False):
         'DatabaseHost': match.group('host'),
         'DatabaseName': match.group('dbname'),
     }
+
+    if 'extra' in match.groupdict() and match.group('extra'):
+        extra = match.group('extra').lstrip('?')
+        import urlparse
+        jdbc_params = urlparse.parse_qs(extra)
+        if 'sslmode' in jdbc_params:
+            use_ssl = ['verify-ca', 'verify-full',
+                       'allow', 'require', 'prefer']
+            sslmode = jdbc_params['sslmode']
```

Comment:
Since the output of `parse_qs` outputs a dict of [String, List], I don't think the if statement below will ever evaluate to true:
```
>>> qs = 'foo=bar&baz=qux'
>>> parse_qs(qs)
{'foo': ['bar'], 'baz': ['qux']}
>>>
```

`sslmode` will always be an array of strings, not a string, so when checking for membership in `use_ssl`, it will always fail.

Instead, I would suggest doing:
```
sslmode = jdbc_params['sslmode']
if sslmode and sslmode[0] in usessl:
   config.update({...})
```